tests/services/application.service.spec.ts

```diff
@@ -1,18 +1,39 @@
 import { ApplicationService } from '../../src/app/core/services/application.service';
 
-const applicationService = new ApplicationService(null);
+const upToDateAppService = new ApplicationService(null);
```

Comment:
Accidentally missed this out, but it seems that this is not used anywhere in the file, so we can afford to remove it as a result. 

Other than that, nothing from my side, thanks!